pydoctor/templatewriter/util.py

```diff
@@ -2,6 +2,7 @@
 
 import warnings
 import collections
+import collections.abc
```

Comment:
```suggestion
import collections.abc
```

Reply:
We need both, actually

Reply:
no you don't

Reply:
see https://snarky.ca/unravelling-the-import-statement/

```python
import collections.abc
```

desugars to :
```python
collections = __import__("collections.abc", globals(), locals())
```

and so is equivalent to:

```python
import collections
import collections.abc
```

Reply:
I don’t think we can import the OrderedDict from collections.abc. But maybe ?

Reply:
```
Python 3.9.7 (default, Sep  9 2021, 23:20:13) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import collections.abc
>>> collections.OrderedDict
<class 'collections.OrderedDict'>
>>> 
```

Reply:
It’s probably not required to use OrderedDict, actually. A normal dict should do the job. So I’ll change that

Reply:
@tristanlatr no I mean `import collections.abc` automatically does a `import collections` so you don't need both lines

Reply:
Ok I see. Thanks for this enlightenment.

Reply:
https://github.com/asottile/reorder_python_imports#removes-duplicate-imports is a tool to automate this